resources/views/components/forms/inputs/file.blade.php

```diff
@@ -0,0 +1,22 @@
+<div>
+    <label for="file" style=" padding: .5rem; border-radius: 5px;" class="{{$buttonContainerClass}}">
```

Comment:
This `style` attribute makes the look of the component very opinionated, and css properties set in the `style` attribute can't be overridden by other CSS. So this locks the component into a single visual style. It's probably better than default browser file-inputs, but I think the purpose of this package is to provide a set of *functional* components, so ideally they wouldn't have any styling at all, or at least very easily configurable styling!